src/app/public/modules/select-field/types/select-field-custom-picker.ts

```diff
@@ -8,6 +8,13 @@ import {
 
 export interface SkySelectFieldCustomPicker {
 
+  /**
+   * Specifies a function to call when users select the text field or button.
+   * @param pickerContext A `SkySelectFieldPickerContext` object that provides values to the
```

Comment:
The parameters are not displaying when I preview the docs. What do we need to tweak to make them display? I tried to emulate other places where we use `@param`.

Reply:
@Blackbaud-SteveBrush, am I doing something wrong here with `@param`? It looks like I implemented it the same here as in other places where it works, but let me know if I'm overlooking something. I don't see an issue for this in the `docs-tools` repo, but let me know if I need to file something.

Reply:
I have a pull request in docs-tools to address this. Stay tuned.

Reply:
Upgrading `@skyux/docs-tools` should fix this issue.

Reply:
Upgrading `docs-tools` displayed the params.